src/components/App/App.scss

```diff
@@ -1,32 +1,76 @@
 @import '../../variables.scss';
 
-.tlbx-body {
-  background: $body-bg;
+.styleguide {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
 }
 
 .tlbx-h2 { text-align: center; }
 
 .styleguide {
+  display: flex;
   display: grid;
   grid-template-columns: 60px 240px 1fr;
   min-height: 100vh;
   max-width: 100%;
+
+  @media only screen and (max-width: 767px) {
+    display: block;
+    overflow: hidden;
+  }
 }
 
 .tlbx-toolbar-wrapper {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  padding: 5px;
+  z-index: 999;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  width: 50px;
```

Comment:
Should be `60px`
<img width="337" alt="screen shot 2017-11-02 at 16 00 50" src="https://user-images.githubusercontent.com/1708450/32332965-09a080d6-bfe7-11e7-9ff9-6d8f992a7d11.png">